Makefile

```diff
@@ -3,14 +3,7 @@ version=1.5.0
 
 default: release
 
-all: release dev alpine pi
-
-dev:
-	docker build -f dev/Dockerfile \
-		--build-arg "SHA=$(sha)" \
-		--build-arg "TAG=$(tag)" \
-		-t $(name) -t nebulouslabs/sia:dev \
-		.
+all: release alpine pi dev
```

Comment:
Did we intend to leave out `debug` and `ci`?

Reply:
I did that intentionally because the script takes a ton of time already and it's running before each push. I'm open to reconsideration, though. Maybe it would be a good sanity check to have those built as well?

Reply:
Maybe rename it from `all` if it's not actually building `all`, or add another target that runs before every push

Reply:
I'll just add them. If it becomes too long I'll reduce the set.